src/snapshot/tosvg.js

```diff
@@ -71,12 +71,19 @@ module.exports = function toSVG(gd, format) {
     svg.selectAll('text')
         .attr('data-unformatted', null)
         .each(function() {
-            // hidden text is pre-formatting mathjax, the browser ignores it but it can still confuse batik
             var txt = d3.select(this);
+
+            // hidden text is pre-formatting mathjax,
+            // the browser ignores it but it can still confuse batik
             if(txt.style('visibility') === 'hidden') {
                 txt.remove();
                 return;
             }
+            else {
+                // force other visibility value to export as visible
```

Comment:
This also makes sense. Presumably it overwrites `visibility: inherit` (or anything that's not `hidden`, for that matter), with `visible`. Which seems correct.